README.md

```diff
@@ -8,11 +8,12 @@ Probably, it does not make sense to use this project with large databases. Howev
 
 ## Supported Databases
 
-| Database  | Image        | Status | Upgrade Support |
-| --------- | ------------ | :----: | :-------------: |
-| postgres  | >= 12-alpine |  beta  |       ✅        |
-| rethinkdb | >= 2.4.0     |  beta  |       ❌        |
-| ETCD      | >= 3.5       | alpha  |       ❌        |
+| Database    | Image        | Status | Upgrade Support |
+|-------------|--------------|:------:|:---------------:|
+| postgres    | >= 12-alpine |  beta  |        ✅        |
+| rethinkdb   | >= 2.4.0     |  beta  |        ❌        |
+| ETCD        | >= 3.5       | alpha  |        ❌        |
+| meilisearch | >= 1.2.0     | alpha  |        ✅        |
```

Comment:
It looks like this table needs to be reformatted.

Reply:
Done.